app/src/main/java/com/github/sdpteam15/polyevents/user/ProfileObject.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.sdpteam15.polyevents.user
+
+import com.github.sdpteam15.polyevents.user.UserObject.CurrentUser
+
+/**
+ * Application user profile constants
+ */
+object ProfileObject{
+    private var default : ProfileInterface? = null
+
+    /**
+     * Defaut profile of the application
+     */
+    val Defaut : ProfileInterface
+        get() {
+            if(default == null)
+                default = Profile("default")
+        return default as ProfileInterface
```

Comment:
Typo : "Default" instead of "Defaut" in english

Reply:
true, it is changed